fearless/Modules/Staking/StakingMain/StakingMainPresenter.swift

```diff
@@ -66,7 +67,8 @@ final class StakingMainPresenter {
                     with: networkStakingInfo,
                     chain: chain,
                     minNominatorBond: commonData?.minNominatorBond,
-                    priceData: commonData?.price
+                    priceData: commonData?.price,
+                    viewIsExpanded: networkInfoViewIsExpanded ?? true
```

Comment:
I would put this just as separate method in view protocol, because it doesn't depend on data. Imagine situation when one has slow connection and open's the screen. The settings will not be applied until network data received